R/mod_review_form_tbl.R

```diff
@@ -69,26 +69,26 @@ mod_review_form_tbl_server <- function(
         form_review_data(), 
         form = form, 
         form_items = form_items,
-        active_subject = active_subject(),
+        active_subject = if(identical(session$userData$review_type(), "form")) NULL else active_subject(),
         pending_form_review_status = NULL,
         is_SAE = identical(title, "Serious Adverse Events")
       )
     }) |> 
-      bindEvent(form_data(), form_review_data(), active_subject())
+      bindEvent(form_data(), form_review_data(), active_subject(), session$userData$review_type())
     
     ############################### Observers: #################################
     
-    
     observe({
       golem::cat_dev(form, "| Resetting userData\n")
       reload_data(reload_data() + 1)
       datatable_rendered(NULL)
       session$userData$pending_form_review_status[[form]] <- NULL
       session$userData$pending_review_records[[form]] <- data.frame(id = integer(), reviewed = character())
-    }) |> 
-      bindEvent(active_subject(), form_review_data(), form_data())
+    }, priority = 1) |> 
```

Comment:
Priority needed to be set, otherwise sometimes this would run AFTER the data was rendered again, which would set datatable_rendered to NULL unintentionally.

Reply:
Do you want to do something like set it to 100? Should always be the first thing to trigger.

Reply:
Sure, we can do that
```suggestion
    }, priority = 100) |> 
```